### ISSUE TITLE
Stuck on "Waiting for skype to initialize"

### ISSUE THREAD

**Issue by tjbcg** [ShyykoSerhiy/phantom-skype#6]
Everything is in the title, and it happened yesterday and did not get it to work since.

Log here : http://pastie.org/10414606

**Comment by ShyykoSerhiy** [comment]
Yeah, looks like guys from MS have changed smth. I'll have a look. Meanwhile, you can try this one
https://github.com/ShyykoSerhiy/skyweb . It's written without need of phantomjs and probably might be more 'stable'(if we can even name those two projects stable) that phantom-skype.

**Comment by wwwgremlin** [comment]
+1 to this issue
